<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>listDriverForm</class>
 <widget class="QDialog" name="listDriverForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Список водителей</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTableWidget" name="tableDriver">
     <property name="minimumSize">
      <size>
       <width>379</width>
       <height>0</height>
      </size>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>100</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>№</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ФИО водителя</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Дата рождения</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="newDriver">
       <property name="text">
        <string>Новый водитель</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editDriver">
       <property name="text">
        <string>Редактировать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteDriver">
       <property name="text">
        <string>Удалить водителя</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
